Gulpfile.js

```diff
@@ -183,6 +194,25 @@ gulp.task('watch', function() {
   gulp.watch('src/js/ripple/*', [ 'build-debug' ]);
 });
 
+// To use this, each javascript file must have /* @flow */ on the first line
+gulp.task('typecheck', function() {
```

Comment:
I agree with Alan that we should have the flow you just described documented somewhere. A doc in the docs folder makes sense, maybe a BUILD.md or something similar